rpm/xulrunner-qt5.spec

```diff
@@ -20,8 +20,8 @@
 %define system_libwebp      1
 
 
-%global mozappdir     %{_libdir}/%{name}-%{milestone}
-%global mozappdirdev  %{_libdir}/%{name}-devel-%{milestone}
+%global mozappdir     %{_libdir}/%{name}-%{greversion}
+%global mozappdirdev  %{_libdir}/%{name}-devel-%{greversion}
```

Comment:
Is the idea here to ensure the app directories don't change when running an alpha version?

Reply:
I tried to append 'a' to the milestone and got packaging issue, so `%{_libdir}/%{name}-devel-%{greversion}` was correct for me. I can't claim that the build system really creates an alpha build. Will test it and come back with feedback.

Reply:
Thanks for explaining. In normal use it looks like this change has no effect, so if that works for you, then I'm happy with it.

Reply:
I confirm that the clean build with extra 'a' character in `%{milestone}` succeeds.